### ISSUE TITLE
Not working with Meteor 1.3.3.1

### ISSUE THREAD

**Issue by kachkaev** [gadicc/meteor-hmr#121]
## Description of Problem

I upgraded to 1.3.3.1 and noticed a strange error: the result of imports became undefined. E.g.

<details>
<summary>`imports/fortytwo.js`</summary>
```js
export const fortytwo = 42;
```
</details>

<details>
<summary>`server/index.js`</summary>
```js
import { Meteor } from 'meteor/meteor';
import { fortytwo } from '/imports/fortytwo.js';

Meteor.startup(() => {
  console.log('fortytwo is ' + fortytwo); // -> fortytwo is undefined
});
```
</details>

The problem with imports occurs only after switching to hmr, all is fine by default.

## MWE

https://github.com/kachkaev/meteor-modules-test

## Environment

* Meteor version: 1.3.3.1
* ecmascript-hot version: 2.0.0-beta.5
* Operating System: Ubuntu

**Comment by kachkaev** [comment]
For those who is facing the same issue and wants to temporary remove meteor-hmr instead of rolling back to 1.3.3:

```bash
meteor remove gadicc:ecmascript-hot
meteor add ecmascript
npm rm --save-dev babel-preset-meteor
rm .babelrc
```

**Comment by gadicc** [comment]
Thanks for the detailed report.  I haven't had a chance to test this exact case yet, but Meteor 1.3.3.1 is working fine so far for me with the latest code base, which I just published now as `gadicc:ecmascript-hot@2.0.0-beta.6`.  Could you try that one and see if it behaves any differently?

Btw, since Meteor 1.3.3 we don't need the babel-preset-meteor anymore... and Meteor works with `.babelrc` on it's own.  It's safe to leave the `react-hot-loader` in your `.babelrc` even without meteor-hmr, even though in this case it won't do that much.

**Comment by Vincent-P** [comment]
Yes `README.md` definitely needs an update.
